psyneulink/_typing.py

```diff
@@ -0,0 +1,19 @@
+from sys import version_info
+
+if version_info < (3, 8):
+    from typing_extensions import Literal
+else:
+    from typing import Literal
+
+from beartype.typing import (
+    Union as Union,
+    Type as Type,
+    Callable as Callable,
+    Optional as Optional,
+    Any as Any,
+    Tuple as Tuple,
+    List as List,
+    Dict as Dict,
+    Iterable as Iterable,
+    Set as Set,
```

Comment:
Why can't these be just imported from typing?

Reply:
See https://github.com/beartype/beartype#pep-585-deprecations. This silences a warning from beartype. Apparently most of typing is slated for deprecation in a few years. Things like List and Tuple can be replaced with list and tuple in python 3.9 but since we are supporting <3.9 we can't use these. I think long term we can replace these when we drop support for 3.8.

Reply:
so the code now uses `_typing` instead of `beartype.typing`?
Can't the same be achieved by just `import beartype.typing as typing` in pnl `__init__.py` ?
if not can all custom PNL types be moved in this file?

Reply:
Yeah, the _typing.py I was thinking could be a place to consolidate typing stuff down the line. I created it mainly for the typing_extensions import of Literal for 3.7 actually. But I think we could move other stuff there as well.